app/registration/mappers/createPatientRequestMapper.js

```diff
@@ -75,15 +78,18 @@ Bahmni.Registration.CreatePatientRequestMapper = (function () {
         return identifiers;
     };
 
-    var setAttributeValue = function (attributeType, attr, value) {
-        if (attributeType.format === "org.openmrs.Concept") {
-            attr.hydratedObject = value;
-        } else if(value === "" || value === null || value === undefined) {
-            attr.voided = true;
-        } else {
-            attr.value = value.toString();
-        }
-    };
+  var setAttributeValue = function (attributeType, attr, value) {
+    if (attributeType.format === "org.openmrs.Concept") {
+      attr.hydratedObject = value;
+    }
+    else if (attributeType.format === "org.openmrs.util.AttributableDate") {
+      attr.value = moment(value).format(DATETIME_FORMAT);
+    } else if (value === "" || value === null || angular.isUndefined(undefined)) {
+      attr.voided = true;
+    } else {
+      attr.value = value.toString().slice(0,50); // person_attribute_type.value is varchar(50), maybe we should validade before.
```

Comment:
Input should validate. I'm creating an issue for this so that we can address in the future.